src/api/index.js

```diff
@@ -292,3 +292,9 @@ export const fetchEdgeImages = ({
     `${EDGE_API}/images?limit=${limit}&offset=${offset}&sort_by=${sortColunm}`
   );
 };
+
+export const getEdgeStatus = (ids) => {
+  return Promise.allSettled(
```

Comment:
Be carefull, this is supported only in browsers older than 2 years (Edge implemented it last year - Jan 2020)

Reply:
So should I use `Promise.all` instead or is it a better alternative?

Reply:
You could use a polyfill - https://github.com/RedHatInsights/user-preferences-frontend/pull/31/files#diff-379120de71991cfea15f15a1225d1ba83a7a50e55a356c831f5f5e0be305ce0b